### PR TITLE
Updating the docs to include rankerType field in QueryDTO

### PR DESCRIPTION
**Pull request by rokulka**

### CODE REVIEW

File: specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json [hunk 1]
```diff
@@ -357,6 +357,10 @@
             }
           ]
         },
+       "rankerType": {
+            "type": "string",
+            "description": "Optional field. Set to 'QuestionOnly' for using a question only Ranker.",
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json]: Please remove trailing comma.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Can you please also confirm if this change is already live?

File: specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json (hunk 1 above)
Comment (rokulka) [reply specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json]: @jhendrixMSFT  fixed this. Yes, this change is live.

**Comment by rokulka** [comment]
> Can you please also confirm if this change is already live?

Yes, the changes are already live.

**Comment by rokulka** [comment]
@jhendrixMSFT Please merge the changes. Thanks.

**Comment by jhendrixMSFT** [comment]
ARM needs to approve the change before I can merge.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**